src/Cake.Common/Tools/GitReleaseNotes/IssueTracker.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Cake.Common.Tools.GitReleaseNotes
+{
+    /// <summary>
+    /// The IssueTracker to be used.
+    /// </summary>
+    public enum IssueTracker
```

Comment:
Yes. All alias's namespaces  are added when imported, names are basically global and currently this usages is pretty "local".